ResearchKit/Charts/ORKFloatRange.h

```diff
@@ -37,52 +38,63 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- The `ORKRangedPoint` class represents a ranged point used in a graph plot.
+ The `ORKFloatRange` class represents a float range. It can be used in graph chart plots to draw a
+ value ranges.
  */
 ORK_CLASS_AVAILABLE
-@interface ORKRangedPoint : NSObject
+@interface ORKFloatRange : NSObject
```

Comment:
Agree with you, `double` is definitely better, sounds less `CoreGraphics` related.